src/parser/mod.rs

```diff
@@ -8715,7 +8715,7 @@ impl<'a> Parser<'a> {
                     offset = Some(self.parse_offset()?)
                 }
 
-                if dialect_of!(self is GenericDialect | MySqlDialect | ClickHouseDialect)
+                if dialect_of!(self is GenericDialect | MySqlDialect | ClickHouseDialect | SQLiteDialect)
```

Comment:
Something else we have been discussing is moving from this `dialect_of` macro to a method on the `Dialect trait` instead (the current code uses both patterns)

So it would look something like
```rust
if self.dialect.supports_limit_comma() ...
```

The benefits are that user defined Dialects can control the behavior and that there is a better place for documenting the behavior

Maybe something to think about for a future PR

Reply:
Makes sense to me! Super easy, so will include this in the next commit to this PR.